bin/setup.js

```diff
@@ -15,7 +15,15 @@ if (saveRequested) {
   var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
   packageJson.scripts = packageJson.scripts || {}
-  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  if (packageJson.scripts[ 'start' ]) {
+    log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
+    ', you can start hoodie with "npm run start-hoodie" instead')
+
+    packageJson.scripts[ 'start-hoodie' ] = packageJson.script[ 'start' ]
+  } else {
+    packageJson.script[ 'start' ] = 'hoodie'
```

Comment:
as Charlotte mentioned above, this should be `packageJson.scripts[ 'start' ] = 'hoodie'`